scripts/cibuild

```diff
@@ -54,7 +54,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
                   -f "${DIR}/../docker-compose.yml" \
                   -f "${DIR}/../docker-compose.airflow.yml" \
                   -f "${DIR}/../docker-compose.test.yml" \
-                  build airflow
+                  build airflow airflow-worker
```

Comment:
On second thought, "add the .env file which we already pull down in `cibuild`" is a way lighter-weight solution than "add another container to the build process" :man_facepalming: